plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -1005,6 +1008,30 @@ public void testUnsignedTypes()
                 .execute(getQueryRunner(), mysqlCreateAndInsert("tpch.mysql_test_unsigned"));
     }
 
+    @Test
+    public void testEnum()
+    {
+        SqlExecutor jdbcSqlExecutor = mySqlServer::execute;
+        jdbcSqlExecutor.execute("CREATE TABLE tpch.test_enum(id int, enum_column ENUM ('b','a','C'))");
+        jdbcSqlExecutor.execute("INSERT INTO tpch.test_enum(id,enum_column) values (1,'a'),(2,'b')");
```

Comment:
Please insert `(3,null)` row and use `assertUpdate()` to improve the test coverage.

Also, add a below code comment
```
// Define enum values in an order different from lexicographical
```

Reply:
> Please insert (3,null) row and use assertUpdate() to improve the test coverage.

Reminder